google_analytics/utils.py

```diff
@@ -60,6 +60,8 @@ def build_ga_params(
 
     # determine the referrer
     referer = referer or request.GET.get('r', '')
+    if referer == domain:
```

Comment:
This test should be that the domain is the same. In other words, `http://test.com/home/` vs `test.com` have the same domain, but this equality test will fail. At the moment `referrer` is coming through as the full URL.

Reply:
As suggested, consider using [urlparse](https://docs.python.org/2/library/urlparse.html#urlparse.urlparse) to get the `netloc` and compare that.